src/EVM/SymExec.hs

```diff
@@ -508,6 +508,7 @@ getExpr solvers c signature' concreteArgs opts = do
     - 0x51: If you call a zero-initialized variable of internal function type.
 
   see: https://docs.soliditylang.org/en/v0.8.6/control-structures.html?highlight=Panic#panic-via-assert-and-error-via-require
+  NOTE: does not deal with e.g. `assertEq()`
 -}
 checkAssertions :: [Word256] -> Postcondition s
```

Comment:
Is this used anywhere? can we either fix it or delete it?

Reply:
This is used in checkAssert which is used everywhere in test.hs (100+ use-cases). It is also used as the primary way that we run `hevm symbolic --code CODE`.

I don't think this code is necessarily wrong? It picks up the standard issues that one likes to pick up without those in `cheatActions`, such as `asssertEq`. Or what do you think we should do instead?